doto-backend/index.js

```diff
@@ -4,12 +4,30 @@ const cors = require('cors')
 const app = express()
 const apiPort = 3000
 
+// Mongoose connection
+const mongoose = require('mongoose');
+const connectionString = "mongodb+srv://701Group2:SoftwareEngineering701Group2@cluster0-ahuai.mongodb.net";
+mongoose.connect(connectionString, { useNewUrlParser: true });
+const db = mongoose.connection;
+
+// Checking for DB connection
+db.once('open', function(){
+    console.log("Connected to MongoDB.");
+});
+db.on('error', function(){
+    console.log(err);
+});
+
 app.use(bodyParser.urlencoded({ extended: true }))
 app.use(cors())
 app.use(bodyParser.json())
```

Comment:
Looks good Lucy. But now express has a builtin body parser which you can call the same way you're currently doing, but replace the "bodyParser" bits with "express" instead. So this means we don't have to import the body parser dependency either.